aea/aea.py

```diff
@@ -48,6 +48,7 @@ def __init__(self, name: str,
                  loop: Optional[AbstractEventLoop] = None,
                  timeout: float = 0.0,
                  debug: bool = False,
+                 programmatic: bool = True,
```

Comment:
I prefer more self explaining naming (`is_programmatic` in this case).

Reply:
Good point. I will open an issue as we should change it systematically for all variables of boolean type.